packages/types/src/key.ts

```diff
@@ -14,3 +14,14 @@ export type PublishableKeyOrFrontendApi =
       frontendApi: string;
       publishableKey?: never;
     };
+
+// export type SatelliteApp =
```

Comment:
@panteliselef this comment?

Reply:
needs to be removed, correct types for developers should be a concern for later